modules/sdk-core/src/index.ts

```diff
@@ -1,6 +1,8 @@
 export * from './account-lib';
+export * as accountLib from './account-lib';
 export * from './api';
 export * from './bitgo';
+export * as bitgo from './bitgo';
```

Comment:
are these still necessary if we're `exporting * from './bitgo` ? Do we still need to namespace them?

Reply:
Not anymore. I'll remove them.